examples/IMU/imu_rotation_vector.py

```diff
@@ -7,11 +7,11 @@
 
 device = dai.Device()
 
-imuVersion = device.getConnectedIMUVersion()
+imuType = device.getConnectedIMUVersion()
```

Comment:
how could this work? I think the api is `getConnectedIMU()`

Reply:
I think this is a bug in github, it shows an older version of commit